src/Cli/Program.cs

```diff
@@ -29,6 +29,8 @@ public static int Main(string[] args)
                 }
             );
 
+            DotNetEnv.Env.TraversePath().Load();
```

Comment:
Do we need to handle cases where the folder traversal results in 'access denied?' so we can gracefully handle and display such an error? This concern is also applicable to the case where a dev provides a full path to the file.

Reply:
> Do we need to handle cases where the folder traversal results in 'access denied?' so we can gracefully handle and display such an error? This concern is also applicable to the case where a dev provides a full path to the file.

That is something I would expect `DotEnv` to handle for us and not something you'd need to handle in your own code.

Reply:
I figured as much. After poking around their code, I don't think `DotEnv` handles all file IO errors, an example being Get: https://github.com/tonerdo/dotnet-env/blob/3dbee19770951a75785d398d02d7eed92261d0fe/src/DotNetEnv/Env.cs File IO not wrapped in try/catch so whatever access errors that may occur are bubbled up to our code.

Reply:
Wouldnt we need to add try catch within our code for such cases? @abhishekkumams, Is this comment resolved?

Reply:
Thanks everyone for the concern. We have moved out of traversePath and now using exact path (DotNetEnv.Env.Load()), which will not throw any IO exception as it is only using `File.Exist()`. Since, no exception would be thrown we do not need to wrap it around try/catch.